hrm/department_app/models/employee_model.py

```diff
@@ -0,0 +1,39 @@
+"""Django models"""
+from django.db import models
+from department_app.models import department_models
+from djmoney.models.fields import MoneyField
+
+class Employee(models.Model):
+    """Employees"""
+    name = models.CharField('Name',max_length=50,primary_key=True)
+
+    department = models.ForeignKey(department_models.Department,on_delete=models.SET_NULL,null=True)
```

Comment:
In accordance with you SRS.md:
Note: If you delete a department, all employees will also be deleted.
If you use on_delete=models.SET_NULL, then employees will not be deleted.